projects/ng-chartist/src/lib/chartist.component.spec.ts

```diff
@@ -75,11 +75,11 @@ describe('chartist component', function(): void {
 
     instance.type = 'Line';
 
-    spyOn(instance, 'renderChart').and.callThrough();
+    spyOn(<any>instance, 'renderChart').and.callThrough();
 
-    instance.update(changes);
+    instance['update'](changes);
```

Comment:
Are these access changes because of the privacy changes?

Reply:
Yes, they are. Otherwise the test fails to transpile.